lib/room.rb

```diff
@@ -1,12 +1,24 @@
-class Rom
+class Room
+  attr_reader :category,
+              :length,
+              :width
```

Comment:
since we never call room.length or room.width, these should not be attr_readers.